docs/architecture/components.mdx

```diff
@@ -92,57 +94,60 @@ The Node Collector is also responsible for recording some k8s metrics, and colle
 
 **Type**: The instrumentor is a Kubernetes operator
 
-**K8s Resource**: Deployment / `odigos-instrumentor`
+**K8s Resource**: `kubectl get deployments -n odigos-system odigos-instrumentor`
 
-**Role**: The instrumentor is responsible for managing the auto instrumentation agents which collect the telemetry data from the selected sources.
+**Role**: The instrumentor is responsible for managing the auto instrumentation agents which collect the telemetry data from the Odigos sources.
 
 - Create and manage the `instrumentationconfig.odigos.io` objects
-  - Triggers the `odiglet` to inspect runtime details of the workload.
+  - Triggers `odiglet` to inspect runtime information of the workload.
   - Triggers the scheduler to deploy the Odigos pipeline.
-  - Merge the Odigos instrumentation rules and odigos config to a per-workload configuration to be used by the agent.
+  - Merge the Odigos instrumentation rules and odigos config to a workload scope configuration to be used by the agent.
 - Inject the instrumentation device into the workload manifest resource as virtual device.
-- Monitor the `odigos-enabled` label and cleanup any instrumentation related resources when a source is removed.
+- Monitor the `source.odigos.io` CRD and cleanup any instrumentation related resources when a source is removed.
 
 ### Odiglet
 
-**Type**: The odiglet is a daemon set and will run one pod per node in the cluster
+**Type**: Odiglet is a daemon set and will run one pod per node in the cluster
 
-**K8s Resource**: DaemonSet / `odiglet`
+**K8s Resource**: `kubectl get daemonsets -n odigos-system odiglet`
 
-**Role**: The odiglet component setups the k8s node environment to run the telemetry-collection agents.
+**Role**: The odiglet component sets up the k8s node environment to run the telemetry-collection agents.
 
-- Inspect the runtime details of the workload and detect the programming language, relevant env variables, and runtime versions.
-- Implement the instrumentation devices. On the presence of the supported devices on the pod, odiglet will mount the agent code to the pod, and add the relevant environment variables for starting it and applying the instrumentation.
-- For eBPF based instrumentation, odiglet will inject the eBPF code into the pod, and read the telemetry data from the eBPF queues to push it downstream.
-- Monitor changes to the instrumentation configs, and update the agents accordingly at runtime.
+- Inspect the runtime details of the instrumented sources containers (programming language, relevant env variables, runtime versions, etc).
+- Implement the Kubernetes virtual instrumentation devices. In the presence of a supported devices on the pod, odiglet will mount the agent code from fily-system to the pod, and add the relevant environment variables to initiate instrumentation.
```

Comment:
```suggestion
- Implement the Kubernetes virtual instrumentation devices. In the presence of a supported device on the pod, Odiglet will mount the agent code from file system to the pod and add the relevant environment variables to initiate instrumentation.
```

is this "from file system" the host's file system, or the pod's?

Reply:
it will mount a directory from the host filesystem into the pod filesystem.

Example: https://github.com/odigos-io/odigos/blob/3cbb11747682f88fa24a7fa08c81ae3f56d59467/odiglet/pkg/instrumentation/instrumentlang/java.go#L58